### DISCUSSION TITLE
Multiple background workers?

### DISCUSSION THREAD

**Discussion by lippoliv** [inventree/InvenTree#5600] [Q&A]
While facing #5594, I was wondering, if I can just spin up multiple worker nodes, so events are processed faster?

**Comment by SchrodingersGat** [comment]
By default, there are 4 worker nodes. This is configurable using the configuration file:

https://github.com/inventree/InvenTree/blob/4e467c75d2c0fbaf6b0cfa82c2a472b40f631d95/InvenTree/config_template.yaml#L167

**Reply by lippoliv** [reply]
OK so probably there's a ENV for this as well, right?

BUT: python is not really multithreading. So my quesion was, can I let docker spin up two dedicated worker instances or will this be a problem?

**Reply by SchrodingersGat** [reply]
You should be able to do this, without issue. But I haven't actually tried it! Let me know how it goes
